### ISSUE TITLE
Support latest version of jupyter lab

### ISSUE THREAD

**Issue by lgeiger** [polyaxon/polyaxon#422]
### Feature motivation

It would be great to be able to run the latest version of Jupyter Lab on Polyaxon.

### Feature description

As noted in https://github.com/polyaxon/polyaxon/issues/310#issuecomment-464721403 currently only Jupyter Lab version `0.33.12` is supported. This is due to an issue which prevents Jupyter Lab from loading the correct CSS file. `0.33.12` is now quite old, and there are already alpha releases for version 1.0.0 available, so it would be great to support the latest versions too.

**Comment by mouradmourafiq** [comment]
It should work if the bug is fixed, you jut need to update your Polyaxonfile to install the version you want to use.

**Comment by lgeiger** [comment]
> It should work if the bug is fixed

Sorry for the confusion. I tried it yesterday with the latest version and the bug is still present. Or am I missing something?

**Comment by mouradmourafiq** [comment]
Can you share your Polyaxonfile to reproduce the bug.

**Comment by lgeiger** [comment]
> Can you share your Polyaxonfile to reproduce the bug.

If I use this file:
```yaml
version: 1

kind: notebook
backend: lab

build:
  image: python:3.7-slim
  build_steps:
    - pip install jupyterlab --no-cache-dir
```
Jupyter Lab will look like this:

![Bildschirmfoto 2019-04-08 um 15 44 29](https://user-images.githubusercontent.com/13285808/55733205-55b13280-5a15-11e9-987f-9c22e4e49f4e.png)

Downgrading to Jupyter Lab version `0.33.12` solves the CSS issue.

Thanks for the help! Polyaxon has been a huge improvement to our deep learning workflow. Keep up the great work!

**Comment by mouradmourafiq** [comment]
I just tried with latest release candidate and it seems the issue is still there, not sure why.

https://github.com/jupyterlab/jupyterlab/issues/4662

**Comment by Derka0** [comment]
The error seems to come from the css path. css loads properly when 'local' is removed in the css path.
Hope it helps.

![Screen Shot 2019-04-26 at 15 19 53](https://user-images.githubusercontent.com/45968685/56810655-1f5a1c80-6837-11e9-8a3d-7c0800a6713b.png)
![Screen Shot 2019-04-26 at 15 20 20](https://user-images.githubusercontent.com/45968685/56810656-1f5a1c80-6837-11e9-8569-70115a1f9731.png)

**Comment by mouradmourafiq** [comment]
@Derka0 did you change the local in browser or did you manage to update Polyaxon to handle that? A PR would be more than welcome.

**Comment by Derka0** [comment]
@mouradmourafiq Unfortunately I changed the path in the browser, I'll do a PR as soon as I have some time

**Comment by Derka0** [comment]
@mouradmourafiq I have been looking at the error, I coudn't quite locate it.
It seems like that a discrepancy occurs between polyaxon notebook spawner and Jupyter Lab when a notebook is created. The Jupyter Lab "themes_url" variable that contain the path to the css variable seems wrong.
Do you have any hint to direct me towards where this can occur?
Thanks.

**Comment by mouradmourafiq** [comment]
@Derka0 unfortunately I have no idea why that path is causing an issue, I will also need to do a deep-dive to explore a possible solution.

**Comment by Derka0** [comment]
It seems that the introduction of the light/dark following the version 0.33 creates the problem. I hope it helps

**Comment by mouradmourafiq** [comment]
There's an issue in the recent versions of jupyterlab for serving and detecting base paths in the IThemesManager plugin, it does not handle `:` for parsing paths, an issue was submitted.

Since this is a blocker for serving any version of JupyterLab, a fix/workaround is made in Polyaxon until that issue is fixed.

Closing this issue for now, the fix will be part of the v0.5 release.
